### PR TITLE
Add 'Windows.UI.Xaml.*' exception types

### PR DESCRIPTION
**Pull request by Sergio0694**
This PR adds the 4 `Windows.UI.Xaml.*` exception types we were missing, and uses a feature switch to throw either those or the `Microsoft.UI.Xaml.*` types. Either group will be fully trimmed depending on the feature switch, so no size impact expected.

### CODE REVIEW

File: src/WinRT.Runtime/ExceptionHelpers.Windows.cs [hunk 1]
```diff
@@ -0,0 +1,118 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using System;
+
+namespace Windows.UI.Xaml
```
Comment (manodasanW) [inline src/WinRT.Runtime/ExceptionHelpers.Windows.cs:6]: Probably want NET guard for WUX version to avoid any netstandard conflicts.

Comment (Sergio0694) [reply src/WinRT.Runtime/ExceptionHelpers.Windows.cs]: Fixed 👍

File: src/WinRT.Runtime/ExceptionHelpers.cs [hunk 2]
```diff
@@ -7,7 +7,6 @@
 using System.Runtime.CompilerServices;
 using System.Runtime.ExceptionServices;
 using System.Runtime.InteropServices;
-using System.Text;
 using WinRT.Interop;
```
Comment (manodasanW) [inline src/WinRT.Runtime/ExceptionHelpers.cs:10]: namespace is used as part of encoding reference below.

File: src/WinRT.Runtime/ExceptionHelpers.Windows.net5.cs [hunk 3]
```diff
@@ -0,0 +1,118 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using System;
```
Comment (manodasanW) [inline src/WinRT.Runtime/ExceptionHelpers.Windows.net5.cs:4]: Can we also add #if NET for this entire file.

**Review by manodasanW — APPROVED** [review]

**Comment by manodasanW** [comment]
CI got far enough to show that tests are now passing.  And previously everything else built fine.

**Pull request merged**